tests/shared/infra/repositories/schedule_repository_mock.test.ts

```diff
@@ -222,9 +264,9 @@ describe('Assert Schedule Repository Mock is correct at all for Class methods',
   })
   it('Should get all classes correctly', async () => {
     const repo = new ScheduleRepositoryMock()
-    const classes = await repo.getAllClasss()
+    const classes = await repo.getAllClasses()
```

Comment:
This method was removed on the last PR so this test will likely have conflitc as well, better delete it